### ISSUE TITLE
rollup-plugin-progress build error

### ISSUE THREAD

**Issue by kevinramharak** [gojutin/typescript-playground-plugin-react#4]
`rollup-plugin-progress` throws an error during build. 

Related issues:
- https://github.com/jkuri/rollup-plugin-progress/issues/19
- https://github.com/jkuri/rollup-plugin-progress/issues/17

Not sure how to fix

**Comment by just214** [comment]
I haven't touched this project in nearly a year, but I'll try to take a look at this when I have time.

**Comment by kevinramharak** [comment]
I was trying to build a file explorer for https://github.com/kevinramharak/playground-browser-fs but I ran into so many build and bundle errors i gave up. No hurries. I can manage with vanilla so far. I was thinking of using your template to implement a vue template as I would prefer that over React.

**Comment by just214** [comment]
Oh, very cool. I am a quite the Vue fan myself, but never got around to a template for it. I did write one in Svelte, but I am guessing it suffers the same issues as the React template:  https://github.com/gojutin/typescript-playground-plugin-svelte

**Comment by kevinramharak** [comment]
I took some time to implement a [Vue 3 template](https://github.com/kevinramharak/typescript-playground-plugin-vue), im not familiar with the whole reactive API but it should be an interesting alternative.

**Comment by just214** [comment]
Very cool! Is it working with the TypeScript playground?

**Comment by kevinramharak** [comment]
Yes its wrapping the playground plugin architecture to implement a Vue 'runtime'. I haven't gotten around to write up a demo, but its a good excuse to prototype with Vue 3

**Comment by just214** [comment]
Please let me know once you write up a demo. I'd love to take a look!
